README.md

```diff
@@ -84,8 +84,15 @@ Please check [here](os.md) for more information about which operating system to
 
 ## Postgres
 Postgres performance can be heavily impacted by not using [LSE](https://github.com/aws/aws-graviton-getting-started/blob/main/c-c%2B%2B.md#large-system-extensions-lse).
-Today, postgres binaries from distributions (e.g. Ubuntu) are not built with `-moutline-atomics` or `-march=armv8.2-a` which would enable LSE. If you're planning to use
-postgres in production, please rebuild it with flags to enable LSE. Note: Amazon RDS for PostgreSQL isn't impacted by this. 
+Today, postgres binaries from distributions (e.g. Ubuntu) are not built with `-moutline-atomics` or `-march=armv8.2-a` which would enable LSE.  Note: Amazon RDS for PostgreSQL isn't impacted by this. 
+
+Update: PostgreSQL started to distribute Ubuntu 20.04 packages optimized with `-moutline-atomics`.
+Please follow [the instructions to set up the PostgreSQL PPA](https://www.postgresql.org/download/linux/ubuntu/) and configure to use the `*-pgdg-testing` channel (we will update these instructions when the packages will be promoted to the stable channel)
+
+```
```

Comment:
we recommend you use these instead of the native ubuntu packages?

Reply:
do we have any idea when they'll move out of testing? perhaps we should wait to merge this until that happens

Reply:
The stable channel has been updated with the binaries built on Nov 18:
https://apt.postgresql.org/pub/repos/apt/dists/focal-pgdg/14/binary-arm64/
I have checked that there are LSE instructions in `objdump /usr/lib/postgresql/14/bin/postgres` for the packages in the stable channel of PostgreSQL PPA.